src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    Figure draw();
```

Comment:
```suggestion
    void draw();
```
why make it return something and then return null? if it's not meant to return anything, make it void instead